Source/WebCore/dom/Position.cpp

```diff
@@ -477,8 +477,10 @@ bool Position::atFirstEditingPositionForNode() const
     case PositionIsBeforeAnchor:
         return true;
     case PositionIsAfterChildren:
-    case PositionIsAfterAnchor:
-        return !lastOffsetForEditing(*protectedDeprecatedNode());
+    case PositionIsAfterAnchor: {
+        auto node = protectedDeprecatedNode();
+        return !lastOffsetForEditing(*node);
```

Comment:
I don't understand this change. It seems it was perfectly safe before? Why use a local variable if it is only used one time?

Reply:
Reverted